components/CardCalendar/src/stories/bem.stories.mdx

```diff
@@ -0,0 +1,1156 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+import { DesignTokenDocBlock } from "storybook-design-token/dist/doc-blocks";
+
+import "../index.scss";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="CSS/Navigation/Card Calendar"
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Design Tokens
+
+<DesignTokenDocBlock categoryName="Card Calendar" viewType="table" />
+
+## Stories
+
+### Default
+
+<Canvas>
+  {/* eslint-disable react/no-unknown-property */}
+  <Story name="Default">
+    <div class="denhaag-card-calendar__wrapper">
+      <div class="denhaag-card-calendar__item">
+        <div class="denhaag-card-calendar__item__header">
+          <span>31 december 2022</span>
+        </div>
+        <div class="denhaag-card-calendar__item__content">
+          <h4 class="denhaag-card-calendar__item__title">
+            <a class="denhaag-card-calendar__item__title__anchor" href="https://www.denhaag.nl">
+              Activity Title
+            </a>
+          </h4>
+          <p class="denhaag-card-calendar__item__description">
+            Hier een korte beschrijving van het evenement in een zin
+          </p>
+          <div class="denhaag-card-calendar__item__footer">
+            <div class="denhaag-card-calendar__item__footer__time">
+              <svg
+                width="14"
+                height="14"
+                viewBox="0 0 14 14"
+                fill="none"
```

Comment:
Volgens mij is deze niet nodig, weet niet zeker.

Reply:
Welke bedoel je? Alleen de fill?